### ISSUE TITLE
Avoid hard-coding links and email addresses

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#670]
**TL;DR**: Avoid hard-coding links and email addresses

**Narrative**
As a developer, I want links to help text, etc. to be stored separately from the code. This makes it more straightforward to have multiple instances and for other people to maintain their own versions.

**Acceptance Criteria**

- [ ] All external links in lookit-api are defined outside the templates/views
- [ ] All email addresses in lookit-api are defined outside the templates/views

**Implementation Notes**
Probably set as environment variables. I could see putting links in the database as being even easier to adjust (then we just add an initial migration to set values) but would go with whatever the standard approach is here.
